src/coreclr/jit/lsra.cpp

```diff
@@ -931,6 +933,7 @@ void LinearScan::setBlockSequence()
             // (i.e. pred-first or random, since layout order is handled above).
             if (!BlockSetOps::IsMember(compiler, readySet, succ->bbNum))
             {
+                JITDUMP("\tSucc block: " FMT_BB, succ->bbNum);
```

Comment:
Don't you need a trailing `\n`?

Reply:
No - I also want to print the sequence list on the same line. Here is the sample JitDump:

```
Current block: BB102
	Succ block: BB103, Criteria: bbNum, Worklist: [BB103 BB104 BB107 BB117 BB118 BB119 BB120 ]
Current block: BB103
Current block: BB104
	Succ block: BB105, Criteria: bbNum, Worklist: [BB105 BB107 BB117 BB118 BB119 BB120 ]
	Succ block: BB106, Criteria: bbNum, Worklist: [BB105 BB106 BB107 BB117 BB118 BB119 BB120 ]
```

While it will definitely be verbose, I am considering to add `LsraDump` and move such logging and more under it.

Reply:
ok, that's great. Thanks for the example.